### PR TITLE
feat(pagination): add aria attributes pagination

### LINKED ISSUE #4134: feat(pagination): add aria-attributes to pagination component
Add aria-attributes:

- `aria-label` to a parent of a pagination list
- `aria-current`, `aria-label` to links in a pagination list
- append additional structure to links in a pagination list, when developer use a link without text
joxi_screenshot_1522239969916

sources: 
http://www.a11ymatters.com/pattern/pagination/
https://getbootstrap.com/docs/4.0/components/pagination/
https://www.w3.org/TR/wai-aria-practices-1.1/#breadcrumb

### PR DESCRIPTION
**Pull request by ghost**
Aria-attributes were added:
- [x] `aria-label`  to a parent of a pagination list ( `nav`)
joxi_screenshot_1522252384132
- [x] `aria-current`, `aria-label` to links (`a class="page-link`) in a pagination list
- [x] appended additional structure to links in a pagination list, when developer use a link without text (like in bootstrap)
joxi_screenshot_1522252221145

closes #4134

### CODE REVIEW

File: src/pagination/pagination.config.ts [hunk 1]
```diff
@@ -8,18 +8,37 @@ export class PaginationConfig {
...
     itemsPerPage: 10,
     boundaryLinks: false,
+    accessibleLinks: false,
+    boundaryAccessibleLinks: false,
     directionLinks: true,
     firstText: 'First',
     previousText: 'Previous',
     nextText: 'Next',
     lastText: 'Last',
+    previousAccessibleText: 'Previous',
```
Comment (IlyaSurmay) [inline src/pagination/pagination.config.ts:19]: 10+ new inputs for accessibility? I think that text for labels should be constructed dynamically from what we have

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: src/pagination/pagination.config.ts (hunk 1 above)
Comment (ghost) [reply src/pagination/pagination.config.ts:19]: It was decided to leave these inputs. But need to add an object which contains these values for adding it to demos

**Comment by onecib** [comment]
@valorkin: Why is this PR not merged ?

**Comment by valorkin** [comment]
Can't recall, but some requested changes wasn't done

**Pull request closed without merging**